cmds/core/kexec/kexec_linux.go

```diff
@@ -90,9 +93,43 @@ func main() {
 		purgatory.Debug = log.Printf
 	}
 
-	if (!opts.exec && flag.NArg() == 0) || flag.NArg() > 1 {
+	var kernelpath string
+	// Regarding potential conflicting options, this might as well be seen
```

Comment:
I don't understand this sentence, what is this trying to say?

Reply:
we will have a kernel path from image info 

we can also have a kernel path from position argument 

first one to be used as default, second one takes precedence if provided.

Reply:
updated the comment a bit to clarify